cfgov/cfgov/settings/base.py

```diff
@@ -642,6 +642,10 @@
 
     # To be enabled when switching the site to use the BCFP logo
     'BCFP_LOGO': {},
+
+    # Improvements to Find A Housing Counselor page
+    # (UI Improvements project, Fall 2018)
+    'HUD_TOOL_IMPROVEMENTS': {},
```

Comment:
FYI, if you think it'd make things easier for you, you could put something here like

```
'HUD_TOOL_IMPROVEMENTS': {'environment is': 'local'},
```

so that way you can run your local server with the `DEPLOY_ENVIRONMENT` environment variable set to `local` (via `.env` or `DEPLOY_ENVIRONMENT=local ./runserver.sh`). This would prevent you (and PR reviewers) from having to manually add the feature flag via the admin each time.

Reply:
Good idea. I've added that to this PR.